lib/flv/transmuxer.js

```diff
@@ -420,6 +421,10 @@ Transmuxer = function(options) {
   packetStream = new m2ts.TransportPacketStream();
   parseStream = new m2ts.TransportParseStream();
   elementaryStream = new m2ts.ElementaryStream();
+  videoCorrectionStream = new m2ts.TSCorrectionStream('video');
```

Comment:
Thinking about this some more, a good name for the stream stage is `TimestampRolloverStream` and then `videoTimestampRolloverStream` et.al. which better describes what this stage does.